src/d_deh.c

```diff
@@ -46,6 +44,13 @@
 
 #include "dsdhacked.h"
 
+#ifndef TRUE
```

Comment:
I think we should rather fix the code to use `true` and `false` instead.

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/859/commits/22e9340eb75c2b02a85ab49b1603a08fb8a2ddb3

Reply:
Thanks!